docs/tutorial.rst

```diff
@@ -151,6 +151,9 @@ Now let's look at the equivalent Hy program:
 
    (simple-conversation)
 
+Note that ``raw_input`` is for Python 2, if you are using Python 3,
```

Comment:
@hylang/core Come to think of it, maybe Hy should alias `input` to `raw_input` under Python 2, kind of like it does `map`.